seatunnel-connectors-v2/connector-kafka/src/main/java/org/apache/seatunnel/connectors/seatunnel/kafka/source/KafkaSourceConfig.java

```diff
@@ -255,10 +255,13 @@ private DeserializationSchema<SeaTunnelRow> createDeserializationSchema(
                     .setCatalogTable(catalogTable)
                     .build();
         }
-
+        boolean ignoreParseErrors =
+                readonlyConfig
+                        .get(MESSAGE_FORMAT_ERROR_HANDLE_WAY_OPTION)
+                        .equals(MessageFormatErrorHandleWay.SKIP);
```

Comment:
I believe it is necessary to add some logging for this action, as it is a risky move